scripts/OpdrachtTwo/Opdracht-2.js

```diff
@@ -0,0 +1,23 @@
+// Opdracht 2a:
+// const userInput = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]');
+// console.log(userInput);
+
+// // Opdracht 2b & 2c:
+// let departmentDescriptionElement= document.getElementById("department-description")
```

Comment:
ik heb eerlijk gezegd geen idee voor dit is gedaan, kun je dit misschien uitleggen ?